src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/PackagePropertyPage.xaml

```diff
@@ -103,6 +103,18 @@
     </StringProperty.DataSource>
   </StringProperty>
 
+  <StringProperty Name="PackageReadmeFile"
+                  DisplayName="README"
+                  Description="The readme document for the package. Supported file formats include only Markdown (.md)."
+                  HelpUrl="https://go.microsoft.com/fwlink/?linkid=2168540"
```

Comment:
This fwlink redirects to a URL that contains `en-us` in it. Please update the fwlink from:

```diff
-https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packagereadmefile
+https://docs.microsoft.com/nuget/reference/msbuild-targets#packagereadmefile
```

Reply:
Updated.